rules/use-code-migrations/rule.md

```diff
@@ -29,59 +29,57 @@ The following assumes you have an existing project with a database context, enti
 
 1. Install EF Core Tools
 
-    ```bash
```

Comment:
Can you revert the indentation on these code blocks (same with the other code blocks)?  Change this to 2 spaces breaks markdown linting

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/2636640/e7e70a11-4682-4402-a097-780326a351d5)
**Figure: Warning caused by incorrect indentation**